prow/plugins/blunderbuss/blunderbuss_test.go

```diff
@@ -22,6 +22,8 @@ import (
 	"sort"
 	"testing"
 
+	"k8s.io/test-infra/prow/plugins"
+
```

Comment:
Let's add tests for the comment handling

Reply:
Yep, we try to delegate to a helper function with scoped inputs as quickly as possible from the registered functions (`handlePullRequestEvent`, `handleGenericCommentEvent` etc). You can see us doing this in other plugins -- unit tests are easier on those.

Reply:
That makes sense. I'll work on doing that.

Reply:
Added tests for both functions.